tests/test_feeds.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (C) 2020 Zeropoint Dynamics
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+
+
+# You should have received a copy of the GNU Affero General Public
+# License along with this program.  If not, see
+# <http://www.gnu.org/licenses/>.
+# ======================================================================
+
+from __future__ import absolute_import
+
+import unittest
+
+from os import path
+
+from zelos import Zelos
+from zelos.feeds import FeedLevel
+
+
+DATA_DIR = path.join(path.dirname(path.abspath(__file__)), "data")
+
+
+class TraceTest(unittest.TestCase):
```

Comment:
___ULTRA___ Nitpick - `TraceTest` -> `FeedTest`?

Reply:
Ah, that's a good catch. Fixed